src/main/java/qna/domain/DeleteHistory.java

```diff
@@ -2,14 +2,40 @@
 
 import java.time.LocalDateTime;
 import java.util.Objects;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
 
+/**
+ * create table delete_history
+ * (
+ *     id            bigint generated by default as identity,
+ *     content_id    bigint,
+ *     content_type  varchar(255),
+ *     create_date   timestamp,
+ *     deleted_by_id bigint,
+ *     primary key (id)
+ * )
+ */
+@Entity
 public class DeleteHistory {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
-    private ContentType contentType;
+
     private Long contentId;
+
+    @Column(columnDefinition = "varchar(255)")
```

Comment:
```suggestion
    @Enumerated(EnumType.STRING)
```

enum의 경우 Enumerated를 사용해볼 수 있습니다 :)